articles/deploying-nodejs-web-app/index.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: engineering-education
+status: publish
+published: true
+slug: deploying-nodejs-web-app
+title: Deploying Your First NodeJS Web App
+description: It’s an introduction to NodeJS and NPM and how to get a basic Express server running to serve static web files. Aimed at front-end developers (who can already create basic websites using HTML, CSS and JS.)
+author: louise-findlay
+date: 2020-07-02T00:00:00-07:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/deploying-nodejs-web-app/hero.JPG
+    alt: header image nodejs
+---
+Congratulations, you’ve finished developing your first NodeJS web app and now you want to publish it on the web. If you’re still in the development process, however, then you may find my **(Link to previous developing NodeJS web app when it gets published)** guide helpful.
+
+<!--more-->
+### Deploying Your First NodeJS Web App
+
+**Add header image**
+
+Congratulations, you’ve finished developing your first NodeJS web app and now you want to publish it on the web. If you’re still in the development process however, then you may find my **(Link to previous developing NodeJS web app when it gets published)** guide helpful.
+
+There are many hosting platforms you can use to deploy your NodeJS web apps such as [Heroku](https://www.heroku.com), [Vultr](https://www.vultr.com), [Linode](https://www.linode.com), [Google Cloud Platform](https://console.cloud.google.com) and [Amazon Web Services](https://aws.amazon.com). We will be using [DigitalOcean](https://www.digitalocean.com) because it’s very popular, simple to use and good value.
```

Comment:
A little nit-picky here, but the last sentence of this paragraph is awkward. Consider revising to something along the lines of "We will be using DigitalOcean because it's very popular, simple to use and provides good value"